README.md

```diff
@@ -3,6 +3,35 @@ Visual Studio Tooling for ASP.NET
 
 Please use the [issue tracker](https://github.com/aspnet/Tooling/issues) to log bugs and participate in discussions related to the ASP.NET Core tooling in Visual Studio. Please be sure to include details on the specific version of Visual Studio that you are using.
 
-To install the latest ASP.NET Core tooling for Visual Studio please install [Visual Studio 2015 ](https://www.visualstudio.com/downloads/visual-studio-2015-downloads-vs).
+The latest released tooling can be downloaded at http://dot.net.
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+# Pre-release builds
+
+If you want to try the latest tooling builds available and provide feedback to the team then you can download them below.
+
+These pre-release builds are internal nightly builds that have had some testing done, but the testing is not as extensive as a released version. There are likely to be bugs, sometimes serious ones, and due to the nature of tooling some of those bugs could force you to rebuild the impacted machine or re-install Visual Studio.
+
+
+| Date     | Link          | Version |
+|----------|---------------|---------|
+| Nightly - 6/7/16 | [DotNetCore.1.0.0.RC3-27-VS2015Tools.Preview2.exe](https://download.microsoft.com/download/A/B/1/AB1D84A0-3E98-43E9-A73E-E55D963162B3/DotNetCore.1.0.0.RC3-27-VS2015Tools.Preview2.exe) | 1.0.20607.27   |
+
+## Installation Instructions
+
+  - Install [Visual Studio 2015 Update 3 RC](https://blogs.msdn.microsoft.com/visualstudio/2016/06/07/visual-studio-2015-update-3-rc/) (Current nightly builds require Update 3 RC to be installed)
+  - Apply workaround for setup issue in VS 2015 U3 RC (There is a known issue with Visual Studio 2015 Update 3 RC, as a result of which, .NET Core Tooling may fail to install after Visual Studio 2015 Update 3 RC is installed. See [here](https://go.microsoft.com/fwlink/?LinkId=808095) for additional details and workaround.)
+  - Install the build downloaded from this page
+
+## Release Notes and Known Issues
+
+- Nightly - 6/7/16
+  - **Notes**
+    - New pre-release .NET Core SDK.
+    - Default web template change to no authentication by default.
```

Comment:
The default authentication for web templates has been changed to **No Authentication**